src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.event;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.common.Category;
+import seedu.address.model.common.Name;
+import seedu.address.model.common.Tag;
+
+
+/**
+ * Represents an event in SOChedule.
+ */
+public class Event {
+    // Fields
+    private final Name name;
+    private final StartDate startDate;
```

Comment:
Instead of having separate class files for startDate and endDate, it might be better to condense into a single Date class. Same for the startTime and endTime